rosidl_generator_py/resource/_msg.py.em

```diff
@@ -175,6 +197,10 @@ class @(spec.base_type.type)(metaclass=Metaclass):
             return False
 @[end for]@
         return True
+
+    @@classmethod
+    def get_fields_and_field_types(cls):
+        return cls._slot_types_dict
```

Comment:
Since this is a getter, should it return a shallow copy so that you can modify the underlying dictionary?

Reply:
Good catch. Fixed.